easybuild/easyblocks/m/mcr.py

```diff
@@ -147,3 +151,12 @@ def make_module_extra(self):
         txt += self.module_generator.set_environment('MCRROOT', os.path.join(self.installdir, self.subdir))
 
         return txt
+
+    def set_subdir(self):
+        if self.subdir: return
+        # determine subdirectory (e.g. v84 (2014a, 2014b), v85 (2015a), ...)
+        subdirs = glob.glob(os.path.join(self.installdir,'v[0-9][0-9]*'))
+        if len(subdirs) == 1:
+            self.subdir = subdirs[0]
```

Comment:
This should be `os.path.basename(subdirs[0])`, since otherwise `module_generator.prepend_paths` will complain about using an absolute path...

Fixed in https://github.com/ComputeCanada/easybuild-easyblocks/pull/1